pkg/vmprovider/providers/vsphere/constants/constants.go

```diff
@@ -56,7 +56,7 @@ const (
 	PCIPassthruMMIOSizeDefault        = "512"
 
 	// MinSupportedHWVersionForPVC is the supported virtual hardware version for persistent volumes.
-	MinSupportedHWVersionForPVC = 13
+	MinSupportedHWVersionForPVC = 15
```

Comment:
Is the min hw version for PVCs not 13? When did it change to 15?

Reply:
With the introduction of CNS support in vSphere 6.7 U3, some changes were made on the ESXi layer as well. Since 6.7U3 shipped with hardware version 15, the required CNS qualification was done with the same version.

Reply:
It would be nice if we can add a comment mentioning this (assuming the info is public).